tools/releaser/breaking_changes/v20240805004.md

```diff
@@ -0,0 +1,36 @@
+# Breaking Changes
+
+## SDK changes
+
+go.mongodb.org/atlas-sdk/v20240805004/admin
+Incompatible changes:
+
+- (*CloudProviderAccessApiService).AuthorizeCloudProviderAccessRoleExecute: changed from func(AuthorizeCloudProviderAccessRoleApiRequest) (*CloudProviderAccessRole, *net/http.Response, error) to func(AuthorizeCloudProviderAccessRoleApiRequest) (*CloudProviderAccessRole, \*net/http.Response, error)
+- (*CloudProviderAccessApiService).CreateCloudProviderAccessRole: changed from func(context.Context, string, *CloudProviderAccessRole) CreateCloudProviderAccessRoleApiRequest to func(context.Context, string, \*CloudProviderAccessRoleRequest) CreateCloudProviderAccessRoleApiRequest
+- (*CloudProviderAccessApiService).CreateCloudProviderAccessRoleExecute: changed from func(CreateCloudProviderAccessRoleApiRequest) (*CloudProviderAccessRole, *net/http.Response, error) to func(CreateCloudProviderAccessRoleApiRequest) (*CloudProviderAccessRole, \*net/http.Response, error)
+- (*CloudProviderAccessApiService).GetCloudProviderAccessRoleExecute: changed from func(GetCloudProviderAccessRoleApiRequest) (*CloudProviderAccessRole, *net/http.Response, error) to func(GetCloudProviderAccessRoleApiRequest) (*CloudProviderAccessRole, \*net/http.Response, error)
+- AuthorizeCloudProviderAccessRoleApiParams.CloudProviderAccessRole: removed
```

Comment:
@lantoli Can you confirm this is expected?

Reply:
we already detected this in the preview, it's a rename from CloudProviderAccessRole to CloudProviderAccessRoleRequestUpdate, e.g. used here: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2556/files